src/core/dev_api/openvino/op/rms_norm.hpp

```diff
@@ -8,20 +8,22 @@
 
 namespace ov {
 namespace op {
-namespace v14 {
+namespace internal {
```

Comment:
Is it required to change namespace?

Reply:
Maybe not required, but I see it as beneficial. We have some other internal ops in the "internal" namespace, so I've updated the namespace to keep consistency and avoid confusion that it's a part of the opset14.